trk/android/app/src/main/AndroidManifest.xml

```diff
@@ -5,7 +5,8 @@
     <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE"/>
     <uses-permission android:name="android.permission.CAMERA"/>
     <uses-feature android:name="android.hardware.nfc" android:required="true" />
-
+    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
```

Comment:
FYI: Activating new Android & Apple permissions requires changes to their respective app store upload builds. That's because the official app page has to reflect these changes to the user.